_posts/2020-07-06-private-bastions.md

```diff
@@ -0,0 +1,180 @@
+---
+layout: post
+title: 'Private Bastion - I'll do what you want me to do'
+date: 2020-07-06
+tags: aws ssm vpc ssh ec2
+author: Paul Symons
+---
```

Comment:
Do you have a default image that can be used?

Reply:
I have added one now